rsconnect/main.py

```diff
@@ -2825,11 +2825,13 @@ def system_caches_list(
     "--language",
     "-l",
     help="The language of the target cache.",
+    required=True,
 )
 @click.option(
     "--version",
     "-V",
     help="The version of the target cache.",
+    required=True,
```

Comment:
The `system_caches_delete()` function claims that `language` and `version` have `Optional[..]`. The `delete_runtime_cache()` function does the same. Should we update them to not have `Optional[..]` typing?